apps/showcase/src/app/rules-engine/rules-engine.template.html

```diff
@@ -32,7 +32,7 @@ <h2 id="rules-engine-example">Example</h2>
       <a href="https://github.com/AmadeusITGroup/otter/blob/main/apps/showcase/src/components/showcase/rules-engine" target="_blank" rel="noopener">Source code</a>
       <ul ngbNav #nav="ngbNav" class="nav nav-tabs mt-3">
         <li ngbNavItem class="nav-item" role="presentation">
-          <a ngbNavLink class="nav-link" [class.active]="activeRuleTab === 'configuration'" (click)="activateRuleTab('configuration')">Configuration rule</a>
+          <a tabindex="0" ngbNavLink class="nav-link" [class.active]="activeRuleTab === 'configuration'" (click)="activateRuleTab('configuration')" (keyup.enter)="activateRuleTab('configuration')">Configuration rule</a>
```

Comment:
We should not need all of this, we should follow the design factory tabs recommendation as it should be accessible.